### ISSUE TITLE
Sending audio messages to iPhone users works sometimes but usually doesn't

### ISSUE THREAD

**Issue by jcollum** [signalapp/Signal-Android#12121]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
I sent a friend several audio messages yesterday. She could play the first one but none of the following messages. I varied the length of the messages, that didn't fix it. Shortest one was 15s. Longest was about 2:30. Confirmed again today that the first (very short) message was playable but the rest were not. 

I am sending audio from Signal to an iPhone user who does not have Signal. I've done this many times in the past (I'd estimate about 8 times) and it's so unreliable that I think Signal should simply not offer to send audio messages to non-Signal users. 

Sending audio to Signal users: works well. Sending audio to non-Signal users: works about 10% of the time. 

If anyone has good luck sending audio to non-Signal users I'd like to know what carrier you are on. 

### Steps to reproduce

- find a friend who has an iPhone but does not use Signal
- send several audio messages of varying length

**Actual result:** one message was playable, the rest were not 
**Expected result:** all messages are playable 

### Screenshots

![Screenshot_20220403-083039~2](https://user-images.githubusercontent.com/704514/161435908-7147e6c2-f6d4-4ff6-b771-2c6a0f3bfdc6.png)
![Screenshot_20220403-083105~2](https://user-images.githubusercontent.com/704514/161435915-5d18055f-72e1-4bce-af73-d2d77fbf0ad4.png)
![Screenshot_20220403-083127~2](https://user-images.githubusercontent.com/704514/161435919-a9b27b2b-f244-4ee9-ac05-46c96cc9452f.png)

### Device info

**Device:** Pixel 3a XL
**Android version:** 12
**Signal version:** 5.34.5

### Link to debug log

https://debuglogs.org/android/5.34.5/dee633c2179b882125b43a1b717e352311c20b4af868d1c8f04606d8bc82eca2

**Comment by jcollum** [comment]
Those screenshots are colossal, is there some option for resizing them via the link?

**Comment by Tursko** [comment]
Although this is sent to an iPhone, it's likely SMS that's the issue. I wonder if it has something to do with the format of the audio message, I'm honestly surprised you can send an audio message to a non-Signal user at all.

**Comment by jcollum** [comment]
It's stock on Android and iOS: https://www.usatoday.com/story/tech/columnist/komando/2021/01/03/how-record-and-send-audio-messages-your-smartphone/4110435001/

**Comment by Tursko** [comment]
Yeah so Signal uses AAC, and iMessage should support AAC. I still wonder if it has something to do with how its being sent (whether that be on the Signal side or carrier side). 

You mentioned that the really small recordings (like 2 seconds) had no issues being played. 
A 2 second clip is around 5 kB, a 15 second clip is around 68 kB... still pretty small in comparison to like a photo. Really weird.

**Comment by jcollum** [comment]
Yeah I really thought I was on to something by sending smaller messages to test out the max length. No dice.

**Comment by jcollum** [comment]
I'm talking to other friends of mine to try and narrow down the scope of the issue.
